lint/linter/test-mdn-urls.ts

```diff
@@ -0,0 +1,115 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import chalk from 'chalk-template';
+import mdnContentInventory from '@ddbeck/mdn-content-inventory';
+
+import { Linter, Logger, LinterData } from '../utils.js';
+import { CompatStatement } from '../../types/types.js';
+
+interface MDNURLError {
+  path: string;
+  ruleName: string;
+  actual: string;
+  expected: string;
+}
+
+const slugs = (() => {
+  const result = new Set<string>();
+  for (const item of mdnContentInventory.inventory) {
+    result.add(item.frontmatter.slug);
+  }
+  return result;
+})();
+
+/**
+ * Get redirects from mdn/content
+ * @returns redirectMap
+ */
+const redirects = await fetch(
+  'https://raw.githubusercontent.com/mdn/content/main/files/en-us/_redirects.txt',
+)
+  .then((res) => res.text())
+  .then((data) => {
+    const lines = data.split('\n');
+    const redirectLines = lines.filter(
+      (line) => line.startsWith('/') && line.includes('\t'),
+    );
```

Comment:
A thought here too: if you wanted to contribute parsing `_redirects.txt` to JSON as a `yarn run content` ([src](https://github.com/mdn/yari/blob/main/tool/cli.ts)) command, then it would be trivial to slurp that up with the content inventory. Then the redirects could be in sync with the slugs and frontmatter.

Reply:
Florian and I discussed this yesterday. Adding the parsing behavior to `@ddbeck/mdn-content-inventory` is more likely to work out in the near term. I'd welcome a PR adding `redirects` to the package ([hereabouts](https://github.com/ddbeck/mdn-content-inventory/blob/fd9e9bb0e1cee084f29f36771fbc820fce4450ab/scripts/generate-inventory.mts#L167-L171)).

Reply:
Wrote a PR: https://github.com/ddbeck/mdn-content-inventory/pull/3